app.js

```diff
@@ -330,8 +330,11 @@ define(function(require) {
 						event.stopPropagation();
 
 						var $this = $(this),
-							$dropdown = $this.parents('th').find('.dropdown-menu-wrapper'),
-							$allDropdowns = container.find('#tasks_review_table .dropdown-menu-wrapper');
+							$dropdown = $this.parents('.column-data').find('.dropdown-menu-wrapper'),
+							$allDropdowns = container.find('#tasks_review_table .dropdown-menu-wrapper'),
+							minHeight = !$dropdown.hasClass('show')
+								? 20 + parseFloat($dropdown.css('height')) + parseFloat(container.find('.footable-header').css('height')) + 'px'
+								: 'unset';
```

Comment:
Here we can take advantage of `jquery#height` to compute the element's height. Also, what about storing those in variables to make it easier to understand what we're doing?

We can use `jquery#css` to un\set values as well, so no need to append `'px'` to our minHeight or use `unset`.
```suggestion
							headerHeight = parseFloat(container.find('.footable-header').height()),
							dropdownHeight = parseFloat($dropdown.height()),
							padding = 20,
							minHeight = $dropdown.hasClass('show')
								? ''
								: headerHeight + padding + dropdownHeight;
```

Reply:
Great advice, thank you!